_opensearch/install/tar.md

```diff
@@ -7,144 +7,455 @@ nav_order: 50
-```
+## (Optional) Test OpenSearch
+
+Before proceeding you should test your installation of OpenSearch. Otherwise, it can be difficult to determine whether future problems are due to installation issues or custom settings you applied after installation. There are two quick methods for testing OpenSearch at this stage:
 
-For other settings, see [Important settings]({{site.url}}{{site.baseurl}}/opensearch/install/important-settings/).
+1. **(Security Enabled)** Apply a generic configuration using the demo security script included in the tar archive.
+1. **(Security Disabled)** Manually disable the security plugin and test the instance before applying your own custom security settings.
 
+The demo security script will apply a generic configuration to your instance of OpenSearch. This configuration defines some environment variables and also applies self-signed TLS certificates. If you would like to configure these yourself, see [Setup OpenSearch in your environment](#setup-opensearch-in-your-environment).
 
-### (Optional) Set up Performance Analyzer
+If you only want to verify that the service is properly configured and will respond to API requests, and you intend to configure security settings yourself, then you may want to disable the security plugin and launch the service without encryption or authentication.
 
-In a tarball installation, Performance Analyzer collects data when it is enabled. But in order to read that data using the REST API on port 9600, you must first manually launch the associated reader agent process:
+An OpenSearch node configured by the demo security script is not suitable for a production environment. If you plan to use the node in a production environment after running `opensearch-tar-install.sh` you should, at a minimum, replace the demo TLS certificates with your own TLS certificates and [update the list of internal users and passwords]({{site.url}}{{site.baseurl}}/security-plugin/configuration/yaml). See [Security configuration]({{site.url}}{{site.baseurl}}/security-plugin/configuration/index/) for additional guidance to ensure that your nodes are configured according to your security requirements.
+{: .warning}
 
-1. Make Performance Analyzer accessible outside of the host machine
+### Option 1: Test your Opensearch settings with security enabled
 
+1. Change to the top directory of your OpenSearch install.
    ```bash
-   cd /usr/share/opensearch # navigate to the OpenSearch home directory
-   cd config/opensearch-performance-analyzer/
-   vi performance-analyzer.properties
+   cd /path/to/opensearch-{{site.opensearch_version}}
    ```
-
-   Uncomment the line `#webservice-bind-host` and set it to `0.0.0.0`:
-
+1. Run the demo security script.
+   ```bash
+   ./opensearch-tar-install.sh
+   ```
+1. Open another terminal session and send requests to the server to verify that OpenSearch is up and running. Note the  use of the `--insecure` flag which is required since the TLS certs are self-signed.
```

Comment:
Delete extra space between "the" and "use." It looks like there are two spaces between the words.

Reply:
Insert comma after the word "flag."

Reply:
Done thank you!